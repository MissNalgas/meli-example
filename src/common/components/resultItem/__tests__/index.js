import ResultItem from '../';
import renderer from 'react-test-renderer';

test('ResultItem renders correctly', async () => {
	const tree = renderer.create(<ResultItem onClick={() => {}} item={{
		picture: 'picture',
		title: 'title',
		price: {
			amount: 3000,
			currency: 'COP'
		},
		city: 'city'
	}}/>).toJSON();

	expect(tree).toMatchSnapshot();

});
