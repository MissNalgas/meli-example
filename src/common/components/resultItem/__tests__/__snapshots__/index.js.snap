// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultItem renders correctly 1`] = `
[
  <button
    className="flex w-full p-small gap-small bg-white items-stretch"
    onClick={[Function]}
  >
    <div
      className="rounded w-image h-image bg-center bg-cover"
      style={
        {
          "backgroundImage": "url('picture')",
        }
      }
    />
    <div
      className="flex flex-col gap-medium justify-center flex-1 items-stretch"
    >
      <div
        className="flex items-center justify-between max-w-2xl"
      >
        <h2
          className="text-2xl"
        >
          $3,000
        </h2>
        <span
          className="text-dark-gray text-xs"
        >
          city
        </span>
      </div>
      <h3
        className="text-lg text-left"
      >
        title
      </h3>
    </div>
  </button>,
  <hr />,
]
`;
